### PR TITLE
Cypress setup

### PR DESCRIPTION
**Pull request by ibanezmartha9**
Cypress setup

### CODE REVIEW

**Review by DiRunner — CHANGES_REQUESTED** [review]
Please delete the examples that cypress creates by default

File: cypress/e2e/1-getting-started/todo.cy.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```
Comment (DiegoCorredorQA) [inline cypress/e2e/1-getting-started/todo.cy.js]: Please delete this file

File: cypress/e2e/2-advanced-examples/actions.cy.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```
Comment (DiegoCorredorQA) [inline cypress/e2e/2-advanced-examples/actions.cy.js]: Please delete it

File: cypress/e2e/2-advanced-examples/aliasing.cy.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/// <reference types="cypress" />
```
Comment (DiegoCorredorQA) [inline cypress/e2e/2-advanced-examples/aliasing.cy.js]: please delete it

File: cypress/support/commands.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (DiegoCorredorQA) [inline cypress/support/commands.js:1]: This file only have comments

File: cypress/support/commands.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+// This example commands.js shows you how to
+// create various custom commands and overwrite
```
Comment (DiegoCorredorQA) [inline cypress/support/commands.js:3]: Only comments

File: cypress/support/e2e.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (DiegoCorredorQA) [inline cypress/support/e2e.js:1]: This only an example

**Review by DiRunner — COMMENTED** [review]
Delete "Getting-started" and advanced examples folders

**Pull request merged**